.github/workflows/build-release.yaml

```
@@ -22,8 +24,18 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           release_branches: main
 
-      - name: Release tag
-        uses: ncipollo/release-action@v1
+      - name: Install Go
+        uses: actions/setup-go@v4
```

actionlint (1.7.12)

[error] the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)